app/config/config.yml.dist

```diff
@@ -204,8 +204,8 @@ thumbnails:
     default_image: [ 1000, 750 ]
     quality: 80
     cropping: crop
-    notfound_image: bolt://app/view/img/default_notfound.png
-    error_image: bolt://app/view/img/default_error.png
+    notfound_image: bolt_assets://img/default_notfound.png
+    error_image: bolt_assets://img/default_error.png
```

Comment:
It was changed to `bolt://` earlier in 3.3, that won't work anymore (but not BC break). The previous value, for 3.2-, is managed [here](https://github.com/bolt/bolt/blob/release/3.3/src/Provider/ThumbnailsServiceProvider.php#L70-L86).